FirebaseAdmin/FirebaseAdmin/Messaging/InstanceIdClient.cs

```diff
@@ -123,21 +122,22 @@ public void Dispose()
             return new TopicManagementResponse(response.Result);
         }
 
-        private void ValidateRegistrationTokenList(List<string> registrationTokens)
+        private void ValidateRegistrationTokenList(IReadOnlyList<string> registrationTokens)
         {
             if (registrationTokens == null)
             {
-                throw new ArgumentNullException("Registration token list must not be null");
+                throw new ArgumentNullException("Registration tokens list must not be null");
```

Comment:
Personally, I think "token" is more fluent, but either is okay.

Reply:
Done